Project_1.cpp

```diff
@@ -15,10 +15,7 @@
         return sf::Color(r, g, b);
     }
 

```

Comment:
There is an excess amount of empty lines that makes the code ugly.
Make sure you don't exceed the default number of empty lines: 1.
Empty lines are used to separate clumps of codes and clumps of functions. It indicates a breathing room for us to understand where code could potentially end or a change in the flow of programming. (e.g. one section would be about setting random color, then the next is to draw the grid. the break is in between setting random colors and drawing the grid).